src/AspNet.Security.OAuth.Imgur/ImgurAuthenticationHelper.cs

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using Microsoft.Framework.Internal;
+using Newtonsoft.Json.Linq;
+
+namespace AspNet.Security.OAuth.Imgur {
+    using System.Globalization;
+
+    /// <summary>
+    /// Contains static methods that allow to extract user's information from a <see cref="JObject"/>
+    /// instance retrieved from Imgur after a successful authentication process.
+    /// </summary>
+    public static class ImgurAuthenticationHelper {
+        public static string GetId([NotNull] JObject user) => GetString(user, "id");
```

Comment:
Could you please bring back the code comments? It would make our documentation task easier. Thanks! :+1: